algorithms.py

```diff
@@ -2275,16 +2287,16 @@ def add_simulator_arguments(parser):
         help="Parameters for the allele frequency trajectory simulation",
     )
     parser.add_argument(
-        "--full-arg",
+        "--all-segments",
         action="store_true",
         default=False,
-        help="Store the full ARG with all recombination and common ancestor nodes",
+        help="Only record edges along coalescing segments.",
```

Comment:
This should be: Record edges for nodes that are coalescent nodes anywhere in the sequence.